CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 # Unreleased
 
+# 5.4.0 (2023.10.09)
+
+- added more validation around optional security. [#2401](https://github.com/stoplightio/prism/pull/2401)
+
```

Comment:
Can we add entries for the other two PRs/fixes/features that are being released too?

The other 2 are:
1. Fixed issue with filling additional properties [#2398](https://github.com/stoplightio/prism/pull/2398)
2. Fixed issue with internal refs inside json schemas [#2402](https://github.com/stoplightio/prism/pull/2402)

Reply:
done!